schema/swagger.yml

```diff
@@ -112,9 +116,9 @@ paths:
       - Accounts
       summary: Requests multisignature groups data
       operationId: getMultisignatureGroups
-      description:
-        By passing in the appropriate options, you can search for a
-        specified account in the system and get its multisignature groups
+      description: |
+        By passing an address of an account, you can search for the specified account in the system.
+        As response, you will get a list of the multisignature groups that this account is member of.
```

Comment:
This is an old documentation phrase `By passing an address of an account` when account  was specified as `address` parameter for the request. Instead focus on `multisignature groups `  description.